### DISCUSSION TITLE
Version 1.18.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#3128] [Changelog]
## Bugfixes
- [#2799](https://github.com/ScratchAddons/ScratchAddons/issues/2799) Words used in addon names and descriptions are inconsistent. (prio-3)
- [#2976](https://github.com/ScratchAddons/ScratchAddons/issues/2976) Firefox containers mess up the extension. (prio-4)
- [#3036](https://github.com/ScratchAddons/ScratchAddons/issues/3036) Scrollbars look different between browsers. (prio-4)
- [#3052](https://github.com/ScratchAddons/ScratchAddons/issues/3052) editor-devtools search disappears when changing language. (prio-4)
- [#3068](https://github.com/ScratchAddons/ScratchAddons/issues/3068) dark-www does not affect the semicolon. (prio-4)
- [#3106](https://github.com/ScratchAddons/ScratchAddons/issues/3106) Addons for project pages run on non-project subpages of project pages. (prio-3)
- [#3116](https://github.com/ScratchAddons/ScratchAddons/issues/3116) editor-stage-left disables dragging palette blocks to other sprites when enabled after load. (prio-4)
- [#3116](https://github.com/ScratchAddons/ScratchAddons/issues/3116) editor-stage-left causes mouse x/y blocks to return wrong value when enabled or disabled after load. (prio-3)
- [#3146](https://github.com/ScratchAddons/ScratchAddons/issues/3146) ScratchAddons.com URLs point to English version. (prio-3)
- [#3152](https://github.com/ScratchAddons/ScratchAddons/issues/3152) infinite-scrolling hides donor text in footer. (prio-4)
- [#3157](https://github.com/ScratchAddons/ScratchAddons/issues/3157) dark-www does not make error modals on studio creation dark. (prio-4)
- [#3165](https://github.com/ScratchAddons/ScratchAddons/issues/3165) editor-theme3 affects reporter bubble. (prio-4)
- [#3165](https://github.com/ScratchAddons/ScratchAddons/issues/3165) editor-colored-context-menus is not compatible with editor-theme3. (prio-4)
- [#3165](https://github.com/ScratchAddons/ScratchAddons/issues/3165) editor-dark-mode does not affect the color picker text. (prio-4)
- [#3165](https://github.com/ScratchAddons/ScratchAddons/issues/3165) editor-theme3 does not affect variable reporters and list reporters. (prio-3)
- [#3178](https://github.com/ScratchAddons/ScratchAddons/issues/3178) editor-devtools uses wrong message on context menu when clean up + gets disabled. (prio-4)
- [#3193](https://github.com/ScratchAddons/ScratchAddons/issues/3193) studio-tools placeholder values are inconsistent. (prio-4)
- [#3196](https://github.com/ScratchAddons/ScratchAddons/issues/3196) Color pickers in settings do not close after clicking. (prio-4)
- [#3204](https://github.com/ScratchAddons/ScratchAddons/issues/3204) scratchr2 makes forum post links scroll to wrong position. (prio-4)
- [#3209](https://github.com/ScratchAddons/ScratchAddons/issues/3209) forum-search crashes on forum signature settings page. (prio-3)
- [#3226](https://github.com/ScratchAddons/ScratchAddons/issues/3226) better-img-uploads button shows on sounds tab when not in English. (prio-4)

## New Features
- [#2682](https://github.com/ScratchAddons/ScratchAddons/issues/2682) New addon: turbowarp-player (prio-3)
- [#2799](https://github.com/ScratchAddons/ScratchAddons/issues/2799) Use extended search in settings. (prio-3)
- [#2986](https://github.com/ScratchAddons/ScratchAddons/issues/2986) New addon: ctrl-enter-post (prio-3)
- [#3044](https://github.com/ScratchAddons/ScratchAddons/issues/3044) New addon: compact-messages (prio-3)
- [#3046](https://github.com/ScratchAddons/ScratchAddons/issues/3046) New addon: default-project (prio-3)
- [#3075](https://github.com/ScratchAddons/ScratchAddons/issues/3075) editor-dark-mode: make speech bubbles dark (prio-3)
- [#3077](https://github.com/ScratchAddons/ScratchAddons/issues/3077) editor-dark-mode: make paint editor dark (prio-3)
- [#3099](https://github.com/ScratchAddons/ScratchAddons/issues/3099) debugger: add feedback link (prio-3)
- [#3104](https://github.com/ScratchAddons/ScratchAddons/issues/3104) editor-devtools: clean up unused local lists in addition to variables (prio-3)
- [#3105](https://github.com/ScratchAddons/ScratchAddons/issues/3105) New addon: customize-avatar-border (prio-3)
- [#3130](https://github.com/ScratchAddons/ScratchAddons/issues/3130) editor-dark-mode: add TurboWarp Dark preset (prio-3)
- [#3144](https://github.com/ScratchAddons/ScratchAddons/issues/3144) New addon: items-per-row (prio-3)
- [#3156](https://github.com/ScratchAddons/ScratchAddons/issues/3156) Add reset dropdown to more types of settings. (prio-3)
- [#3171](https://github.com/ScratchAddons/ScratchAddons/issues/3171) editor-devtools: allow Alt-Click in case middle click is not available (prio-3)

**Comment by FunctionalMetatable** [comment]
> @apple502j:
> * [#3086](https://github.com/ScratchAddons/ScratchAddons/issues/3086) New addon: ctrl-enter-post (prio-3)

Its actually #2986
